src/test/testSchemaGenerator.ts

```diff
@@ -1370,3 +1371,21 @@ describe('Generating a full graphQL schema with resolvers and connectors', () =>
     });
   });
 });
+
+describe('chainResolvers', () => {
+  it('can chain two resolvers', () => {
+    const r1 = (root: number) => root + 1;
+    const r2 = (root: number, { addend }: { addend: number }) => root + addend;
```

Comment:
shouldn't be "added" and not "addend"? or maybe it's a word i'm not familiar with..?